cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -155,6 +157,8 @@ public class KafkaReconciler {
     private String loggingHash = "";
     private final Map<Integer, String> brokerConfigurationHash = new HashMap<>();
     private final Map<Integer, String> kafkaServerCertificateHash = new HashMap<>();
+    private PemTrustSet pemTrustSet;
+    private PemAuthIdentity pemAuthIdentity;
```

Comment:
Same as everywhere else. It should be named based on what it is and not based on the type. Also, why don't we pass these from the outside through the construtor? M(aybe we discussed it on one of the calls, I might not remember anymore :-().

Reply:
I tried to see if I could do the Secret lookup inside KafkaAssemblyOperator and pass them in, but struggled to get the KafkaAssemblyOperator tests to work with that change

Reply:
Might be worth opening a follow-up issue?

Reply:
Sure, I can open one, but I guess the question is whether we would actually do it since it would be quite a complicated task refactoring the tests and perhaps for little gain?